DevEdu.API/DevEdu.Business.Tests/TestDataHelpers/UserData.cs

```diff
@@ -8,6 +8,7 @@ namespace DevEdu.Business.Tests
     public static class UserData
     {
         public const int expectedUserId = 33;
+        public const int idMinimum = 1;
```

Comment:
it's a bad practice to have such values as consts
it's better to var them in each test method separately